### PR TITLE
Update getting_started.rst

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -48,10 +48,10 @@ provider and updating the Django settings for this project with the client crede
 A new OAuth 2.0 client can be created at ``http://127.0.0.1:8000/admin/oauth2/client/``.
 
     1. Click the :guilabel:`Add client` button.
-    2. Leave the user field blank.
+    2. Pick a user.
     3. Specify the name of this service, ``Course Discovery Service``, as the client name.
-    4. Set the :guilabel:`URL` to the root path of this service: ``http://localhost:18381/``.
```
Comment (clintonb) [inline docs/getting_started.rst:53]: Revert. These ports are relevant for Docker, which we should all be using.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -48,10 +48,10 @@ provider and updating the Django settings for this project with the client crede
 A new OAuth 2.0 client can be created at ``http://127.0.0.1:8000/admin/oauth2/client/``.
 
     1. Click the :guilabel:`Add client` button.
-    2. Leave the user field blank.
+    2. Pick a user.
```
Comment (clintonb) [inline docs/getting_started.rst]: Nit: Select a user.

**Review by schenedx — COMMENTED** [review]
I believe the instruction is based on the docker setup. I believe you are using the local machine setup. 
So, I don't know if we should make the change here if we haven't made the decision on which setup we should officially support

**Comment by schenedx** [comment]
@MatthewPiatetsky But I agree, we should change the setup to `select a user`

**Review by clintonb — APPROVED** [review]

**Comment by clintonb** [comment]
Closed due to inactivity.

**Pull request closed without merging**